mina.netty/src/test/java/org/kaazing/mina/netty/NioSocketIT.java

```diff
@@ -71,11 +76,20 @@
 import org.kaazing.mina.netty.socket.nio.DefaultNioSocketChannelIoSessionConfig;
 import org.kaazing.mina.netty.socket.nio.NioSocketChannelIoAcceptor;
 import org.kaazing.mina.netty.socket.nio.NioSocketChannelIoConnector;
+import org.kaazing.test.util.MethodExecutionTrace;
 
 /**
  * Integration test for mina.netty layer
  */
 public class NioSocketIT {
+
+    private final TestRule trace = new MethodExecutionTrace();
+
+    private final TestRule timeout = new DisableOnDebug(new Timeout(15, TimeUnit.SECONDS));
+
+    @Rule
+    public final TestRule chain = outerRule(trace).around(timeout);
```

Comment:
Please alter this to use the new [ITUtil](https://github.com/kaazing/gateway/blob/develop/test.util/src/main/java/org/kaazing/test/util/ITUtil.java) class, method createTimeout. I actually renamed this method to createRuleChain in my PR https://github.com/kaazing/gateway/pull/291 so it might be best to merge that first.